sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -28,6 +28,15 @@ from form documents. It includes the following main functionalities:
 </dependency>
 ```
 [//]: # ({x-version-update-end})
+> Note: This version of the client library defaults to the `v2.1` version of the service.
+
+This table shows the relationship between SDK versions and supported API versions of the service:
+
+|SDK version|Supported API version of service
+|-|-
+|3.0.0 | 2.0
+|3.0.1 | 2.0
+|3.1.0 | 2.1
```

Comment:
I thought Java added multi-service support in their GA, so this should be 2.0 and 2.1